src/transform-sdk/js/package_toolchain.py

```diff
@@ -19,6 +19,7 @@ def download_via_curl(url, file):
     """
     Download by shelling to curl. Simpler than than the pure python approach 🤷
     """
+    print(url)
```

Comment:
nitpick: errant print stmt

Reply:
I actually want it there. This script is only run manually and it gives context to the curl output that happens after.

Reply:
kk, just making sure